unit-tests/syncer/test-ts-desync.py

```diff
@@ -0,0 +1,89 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2021 Intel Corporation. All Rights Reserved.
+
+import pyrealsense2 as rs
+from rspy import log, test
+import sw
+
+
+# The timestamp jumps are closely correlated to the FPS passed to the video streams:
+# syncer expects frames to arrive every 1000/FPS milliseconds!
+sw.fps_c = sw.fps_d = 30
+sw.init( syncer_matcher = rs.matchers.dic_c )
```

Comment:
Remind me, what does it mean to sync dic_c?
If the d , i and c are synced what does it mean to sync this frameset into another color?

Reply:
(TS: (TS: Depth Infrared Confidence) Color)

Reply:
Usually DIC can be matched in a different way (e.g., by frame number) since they'll originate from a different sensor. Technically you're right in that with a match config like the above you may as well just have (TS: D I C Color)...